server/models/users.sql

```diff
@@ -0,0 +1,32 @@
+\c food_direct
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TYPE role AS ENUM ('superadmin','admin', 'user');
+
+CREATE OR REPLACE FUNCTION trigger_set_timestamp()
+RETURNS TRIGGER AS $$
+BEGIN
+  NEW.updated_at = NOW();
+  RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE TABLE users (
+  id uuid NOT NULL DEFAULT uuid_generate_v4(),
+  first_name varchar(255) NOT NULL,
+  last_name varchar(255) NOT NULL, 
+  email varchar(255) NOT NULL UNIQUE,
+  phoneNumber varchar(255),
```

Comment:
you are using camelCase for `phoneNumber` while snake_case for the rest please be consistent.